tests/platform_tests/reboot_timing_constants.py

```diff
@@ -30,7 +30,9 @@
     "COMMON": {
         "PORT_INIT|Start": re.compile(r'.*NOTICE swss#orchagent.*initPort: Initialized port.*'),
         "PORT_READY|Start": re.compile
-        (r'.*swss#orchagent.*updatePortOperStatus.*Port Eth.*oper state set down to up.*'),
+        (r'.*swss#orchagent.*updatePortOperStatus.*Port Eth.*oper state set( from)? (down|up) to up.*'),
+        "PORT_READY|Start-changes-only": re.compile
```

Comment:
Why can't we update and still use only `Start` regex? What's the need for `Start-changes-only`?

Reply:
This is so that in the cases where there are no port oper status changes from down to up (which should be all warm reboots, I think), there is still a start timestamp recorded for when _some_ `updatePortOperStatus` is run, and will correctly populate the timings table at the end.